client/src/components/__mocks__/HandMarkedPaperBallot.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { HandMarkedPaperBallotProps } from '../HandMarkedPaperBallot'
+
+const HandMarkedPaperBallot = ({
+  ballotStyleId,
+  election,
+  precinctId,
+  onRendered,
+}: HandMarkedPaperBallotProps) => {
+  setTimeout(onRendered, 1000)
```

Comment:
Why 1000 if this is just a mock? Why not `setImmediate` or even `useEffect` like a real component would use?

Reply:
I wanted to mimick the real behavior that does take some time. You don't think it's useful?